packages/dataviews/src/components/dataviews/stories/index.story.tsx

```diff
@@ -326,3 +326,23 @@ export const CustomPerPageSizes = () => {
 		/>
 	);
 };
+
+export const CustomEmptyState = () => {
+	const [ view, setView ] = useState< View >( {
+		...DEFAULT_VIEW,
+		fields: [ 'title', 'description', 'categories' ],
+	} );
+	return (
+		<DataViews
+			getItemId={ ( item ) => item.id.toString() }
+			paginationInfo={ { totalItems: 0, totalPages: 0 } }
+			data={ [] }
+			view={ view }
+			fields={ fields }
+			onChangeView={ setView }
+			actions={ actions }
+			defaultLayouts={ defaultLayouts }
+			empty={ <p>Nothing found, add some data to your dataview.</p> }
```

Comment:
Realistic usage will probably look like this

```suggestion
			empty={ view.search ? ( <p>Search returned nothing.</p> ) : ( <p>No data.</p> ) }
```